pipeline/matching/rankers/variety_test.py

```diff
@@ -1,6 +1,6 @@
 import pytest
 
-from pipeline.matching.rankers import VarietyRanker
+from pipeline.matching.rankers import VarietyRanker, variety
```

Comment:
Importing is subjective. This is fine with me, as I don't think we will be adding anything complex to the variety.py file. 

For more complex projects, I would advise against importing the whole file and importing the specific function/class.

For example: 
```
from pipeline.matching.rankers import VarietyRanker, dep_gencheck
```